editor/src/clj/editor/asset_browser.clj

```diff
@@ -430,13 +430,16 @@
                 {:title "Delete Files?"
                  :icon :icon/circle-question
                  :header "Are you sure you want to delete these files?"
-                 :content {:fx/type fxui/label
+                 :content {:fx/type :v-box
                            :style-class "dialog-content-padding"
-                           :text (format "You are about to delete: \n%s"
-                                         (->> selection
-                                              (map #(str "\u00A0\u00A0\u2022\u00A0"
-                                                         (resource/resource-name %)))
-                                              (string/join "\n")))}
+                           :children [{:fx/type fxui/label
+                                       :text "You are about to delete:"}
+                                      {:fx/type fxui/text-area
+                                       :text (format "%s"
+                                                     (->> selection
+                                                           (map #(str "\u00A0\u00A0\u2022\u00A0"
+                                                                      (resource/resource-name %)))
+                                                           (string/join "\n")))}]}
```

Comment:
Looks like there's no need to wrap this in `(format "%s" ...)` now?

Reply:
you are totally right